_posts/2017-04-02-Increasing-Rusts-Reach-2.0.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Increasing Rust’s Reach 2.0"
+author: Ashley Williams
+---
+
+The Rust team is happy to announce that we're running our Increasing Rust's Reach
```

Comment:
should this link to the previous announcement?

Reply:
i link to the previous announcement in the "last year" section but i can do so again here as well :)